next.config.mjs

```diff
@@ -127,10 +133,13 @@ const sentryConfig = {
 // Next.js Configuration with `next.intl` enabled
 const nextWithIntl = withNextIntl('./i18n.tsx')(nextConfig);
 
+// Enables Million.js React Compiler for Next.js
+const nextWithMillion = million.next(nextWithIntl, { auto: true, rsc: true });
```

Comment:
```suggestion
const nextWithMillion = million.next(nextWithIntl, {
  // support react server components
  auto: { rsc: true },
  // We don't want to send telemetry data to Million
  telemetry: false,
});
```

[docs ref for rsc](https://million.dev/docs/install#add-the-compiler-to-your-application)
[docs ref for telemetry](https://million.dev/telemetry#how-do-i-opt-out)